modules/runner/src/integrationTest/java/org/apache/ignite/internal/runner/app/ItIgniteNodeRestartTest.java

```diff
@@ -1421,7 +1421,7 @@ public void createTableCallOnMultipleNodesTest(boolean populateStableAssignments
         }
 
         // Assume that the table id will always be 7 for the test table. There is an assertion below to check this is true.
-        int tableId = 7;
+        int tableId = 8;
```

Comment:
Do we really need to rely on IDs here? If we still need, please introduce a constant.

Reply:
Test uses tableId _before_ the table is created